tooling/ast_fuzzer/src/program/func.rs

```diff
@@ -598,28 +601,59 @@ impl<'a> FunctionContext<'a> {
             return self.gen_for(u);
         }
 
+        if freq.enabled_when("loop", self.budget > 1 && self.unconstrained()) {
+            return self.gen_loop(u);
+        }
+
+        if freq.enabled_when("while", self.budget > 1 && self.unconstrained()) {
+            return self.gen_while(u);
+        }
+
+        if freq.enabled_when("break", self.in_loop) {
+            return Ok(Expression::Break);
+        }
+
+        if freq.enabled_when("continue", self.in_loop) {
+            return Ok(Expression::Continue);
+        }
```

Comment:
```suggestion
        if freq.enabled_when("break", self.in_loop && self.unconstrained()) {
            return Ok(Expression::Break);
        }

        if freq.enabled_when("continue", self.in_loop && self.unconstrained()) {
            return Ok(Expression::Continue);
        }
```

We can't break out of ACIR loops.